src/app/treasure-hunt/treasure-hunt-report/report-graphs/report-graphs.component.html

```diff
@@ -1,29 +1,167 @@
-<div class="row">
-  <div class="print-avoid-break col-12 col-lg-6">
-    <label class="group-label w-100">Current Cost Status ({{settings.currency}})</label>
-    <app-cost-pie-chart [treasureHuntResults]="treasureHuntResults" [isBaseline]="true" [showPrint]="showPrint" [settings]="settings">
-    </app-cost-pie-chart>
-  </div>
-  <div class="print-avoid-break col-12 col-lg-6">
-    <label class="group-label w-100">Projected Cost Status ({{settings.currency}})</label>
-    <app-cost-pie-chart [treasureHuntResults]="treasureHuntResults" [isBaseline]="false" [showPrint]="showPrint">
-    </app-cost-pie-chart>
-  </div>
-</div>
-<div class="row">
-  <div class="print-avoid-break d-flex flex-column justify-content-end"
-    [ngClass]="{'col-lg-3 col-12': !showPrint, 'col-4': showPrint}" *ngFor="let savingsItem of savingsItems">
-    <div class="p-0">
-      <label class="group-label w-100">{{savingsItem.label}} ({{settings.currency}})</label>
-      <app-utility-donut-chart [showPrint]="showPrint" [savingsItem]="savingsItem">
-      </app-utility-donut-chart>
-    </div>
-  </div>
-</div>
-<div class="row">
-  <div class="print-break-before col-12">
-    <label class="group-label w-100">Payback Period vs. Effort to Implement</label>
-    <app-effort-savings-chart class="m-3" [treasureHuntResults]="treasureHuntResults" [showPrint]="showPrint" [settings] = "settings">
-    </app-effort-savings-chart>
-  </div>
-</div>
\ No newline at end of file
+<ng-container *ngIf="!showPrint">
```

Comment:
This whole file - I think you'll be able to  (and should) avoid copy/pasting the html for each of these charts in showPrint wrappers if you:
use *ngIf="graphTab == 'cost' || showPrint" 
along with conditionally applied styles --> [ngClass].